rllib/algorithms/marwil/marwil_tf_policy.py

```diff
@@ -55,7 +55,7 @@ def postprocess_trajectory(
             # Create an input dict according to the Model's requirements.
             index = "last" if SampleBatch.NEXT_OBS in sample_batch else -1
             input_dict = sample_batch.get_single_step_input_dict(
-                self.model.view_requirements, index=index
+                self.view_requirements, index=index
```

Comment:
What's the reason behind this change? Can we get rid of it?

Reply:
I agree this looks strange at first. The reason behind it is: 

- We still need the policies for the RL Module API - in this case the BC policy depends on the MARWIL one
- This policy calls the view requirements of the model - but the `ModelV2`. In the RL Module case `model` is a `RLModule` and does not have the attribute `view_requirements`; this throws an exception.
- To overcome this I call the view requirements from the policy, which should be the same I guess - any exceptions you can think of?

This should become obsolete as soon as we are deprecating MARWIL to `rllib_contrib` as then the BC policy must be independent of the MARWIL one and we do not need any postprocessing as we do not need returns.

Reply:
@ArturNiederfahrenhorst Shall we go with it?

Reply:
Yes, all good. This should indeed not pose a problem.

Reply:
fine with me, too